packages/common/src/ui/components/portals/DetailPanel/DetailPanel.tsx

```diff
@@ -112,7 +112,7 @@ export const DetailPanelPortal: FC<DetailPanelPortalProps> = ({
             />
           </Box>
         </Grid>
-        <Grid item flex={1}>
+        <Grid item flex={1} style={{ overflowY: 'scroll' }}>
```

Comment:
with the pricing and transport sections.. the side panel is getting too big for my screen!